big_tests/tests/graphql_http_upload_SUITE.erl

```diff
@@ -194,6 +221,10 @@ admin_http_upload_not_configured(Config) ->
     ?assertEqual(<<"module_not_loaded_error">>, get_err_code(Result)),
     ?assertEqual(<<"mod_http_upload is not loaded for this host">>, get_err_msg(Result)).
 
+domain_admin_get_url_no_permission(Config) ->
+    Result = admin_get_url(<<"AAAAA">>, <<"test">>, 123, <<"Test">>, 123, Config),
```

Comment:
I think it might be more valuable to test the secondary domain instead, because this way we would be sure that the domain admin can't access a different domain even if it exists - which is actually the main reason to have the domain admin at all.